_includes/api/en/4x/app-get-method.md

```diff
@@ -12,4 +12,19 @@ subsequent routes if there's no reason to proceed with the current route.
 app.get('/', function (req, res) {
   res.send('GET request to homepage');
 });
+
+// path could be array of strings
+app.get(['/foo/bar', '/other/bar'], function (req, res) {
+  res.send('GET request to any of above path');
+});
+
+// path could be regex
+app.get(/^\/yellboom(er|s)?$/, function (req, res) {
+  res.send('GET request to page that starts with startswithboom');
```

Comment:
i.m.o. let's use examples that at least seem plausible to exist in an application. For example, `/^\/user\/(create|[0-9]+)$` at least looks more like something that could exist and help people.